src/pages/Home.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Home() {
+  return (
+    <div className="home">
+      <div class="content">
+        <div className="img-home">
+          <img src="/img/bro-satisfaction.svg" />
+        </div>
+        <div className="home-text">
+          <h3>Bem vindo</h3>
+          <p>
+            A empresa FolhaCPA é uma plataforma para realizar pesquisas de
+            satisfação entre alunos de graduação, pós graduação, mestrado e
+            doutorado.
+          </p>
+
+          <Link className="info-link" to="/Details.js">
```

Comment:
Aqui é a rota, não o nome do componente
```suggestion
          <Link className="info-link" to="/details">
```

Reply:
ok

Reply:
@Euclides-R você não alterou nada

Reply:
@eucleciojosias e como eu faço isso?

Reply:
Agora ta certo